src/s3.ts

```diff
@@ -8,6 +8,7 @@ import { AccessCapability, IAccessSpec, K9PolicyFactory } from './k9policy';
 export interface K9BucketPolicyProps extends s3.BucketPolicyProps {
   readonly k9DesiredAccess: Array<IAccessSpec>;
   readonly encryption?: BucketEncryption;
+  readonly disableEncryptionAtRestMethodCondition?: boolean;
```

Comment:
JSDocs that explain the what it does, but more importantly `@default true`

nit: flags should be named after the positive case. You can even drop the word disable that way, like the one below.

Reply:
Yeah, I agree about naming flags for the positive case, but couldn't convince TypeScript to do the right thing when the property is not defined.  I'll try `@default true`